src/geode/model/helpers/convert_to_mesh.cpp

```diff
@@ -540,7 +550,228 @@ namespace
         return builder.get_result();
     }
 } // namespace
+namespace
+{
+    template < typename Model >
+    void build_edges_from_model( const Model& model,
+        geode::ModelToMeshMappings& model2mesh,
+        std::unique_ptr< geode::EdgedCurveBuilder< Model::dim > >&
+            mesh_builder )
```

Comment:
```suggestion
        geode::EdgedCurveBuilder< Model::dim >&
            mesh_builder )
```
simpler to use directly the object